db/db_impl_compaction_flush.cc

```diff
@@ -2194,6 +2256,8 @@ Status DBImpl::BackgroundCompaction(bool* made_progress,
     manual_compaction->in_progress = true;
   }
 
+  std::unique_ptr<TaskLimiterToken> task_token;
```

Comment:
I assume that you've verified that end of scope never outlives the task limiter.